File: cloud/oauth-server/src/main/java/pers/acp/admin/oauth/base/OauthBaseRepository.java
package pers.acp.admin.oauth.base;

import org.springframework.data.repository.NoRepositoryBean;
import pers.acp.admin.common.base.BaseRepository;

import java.io.Serializable;

/**
 * @author zhangbin by 2018-1-15 16:59
 * @since JDK 11
 */
@NoRepositoryBean
public interface OauthBaseRepository<T, ID extends Serializable> extends BaseRepository<T, ID> {
}

File: cloud/oauth-server/src/main/kotlin/pers/acp/admin/oauth/base/OauthBaseRepository.kt
package pers.acp.admin.oauth.base

import org.springframework.data.repository.NoRepositoryBean
import pers.acp.admin.common.base.BaseRepository

/**
 * @author zhangbin by 2018-1-15 16:59
 * @since JDK 11
 */
@NoRepositoryBean
interface OauthBaseRepository<T, ID> : BaseRepository<T, ID>